app/src/androidTest/java/org/mozilla/focus/activity/CustomTabTest.kt

```diff
@@ -108,13 +107,6 @@ class CustomTabTest {
             openCustomTabMenu()
             clickOpenInFocusButton()
         }
-
-        browserScreen {
-            mDevice.waitForIdle(waitingTime)
-            verifyNumberOfTabsOpened(2)
-            mDevice.pressBack()
-            verifyPageURL(browserPage)
-        }
```

Comment:
The removed steps verify 2 things:
- that the page from the custom tab opened in a new tab and the initial one (see L100) wasn't replaced. 
- that pressing back from the custom tab page shows the other tab (testing an old regression here actually) 
- the floating tabs button was visible when I've tested this manually so it's probably just a timing issue? While running the test I see that for some reason it deletes the tabs instead of verifying the tab counter.

Reply:
Just as a note: This test fails as a regression of https://github.com/jonalmeida/focus-android/commit/12cc22a78884e2d5ce9a4579cee04f628d91667c